SIT.Manager.Avalonia/Services/AkiServerService.cs

```diff
@@ -80,30 +80,36 @@ public string[] GetCachedServerOutput()
 
     public bool IsUnhandledInstanceRunning()
     {
-        Process[] akiServerProcesses = Process.GetProcessesByName(Path.GetFileNameWithoutExtension(SERVER_EXE));
-
-        if (akiServerProcesses.Length > 0)
+        try
         {
-            if (_process == null || _process.HasExited)
-            {
-                return true;
-            }
+            Process[] akiServerProcesses = Process.GetProcessesByName(Path.GetFileNameWithoutExtension(SERVER_EXE));
 
-            foreach (Process akiServerProcess in akiServerProcesses)
+            if (akiServerProcesses.Length > 0)
             {
-                if (_process.Id != akiServerProcess.Id)
+                if (_process == null || _process.HasExited) // error here (No process is associated with this object.)
                 {
                     return true;
                 }
+
+                foreach (Process akiServerProcess in akiServerProcesses)
+                {
+                    if (_process.Id != akiServerProcess.Id)
+                    {
+                        return true;
+                    }
+                }
             }
         }
-
+        catch
+        {
+            return false;
+        }
```

Comment:
These changes aren't needed anymore as the root issue is fixed

Reply:
I don't remember I ever done that I only remember I added try catch. Will fix it.